### ISSUE TITLE
Import Templates Have't Items

### ISSUE THREAD

**Issue by UneasyTheWay** [MantasTumenas/Zabbix-template-for-Microsoft-SQL-Server#13]
After import Template Template Microsoft SQL Server DE Baseline.xml (With SQL instance discovery\Templates), Zabbix have't items

**Comment by MantasTumenas** [comment]
If imported template is seen in host Discovery section and they are shown as Enabled, then you need to wait a little. For some 1 min, for some 15 min. Depends on the server. How your situation is today?

**Comment by UneasyTheWay** [comment]
> If imported template is seen in host Discovery section and they are shown as Enabled, then you need to wait a little. For some 1 min, for some 15 min. Depends on the server. How your situation is today?

Do you need to configure the account password of the database?

**Comment by MantasTumenas** [comment]
No, you do not. But as stated in a documentation - Grant rights for Zabbix Agent service account. By default, Zabbix Agent service account is NT AUTHORITY\SYSTEM which is already in SQL Server

**Comment by ioannis-pan** [comment]
I have the same issue, it worked only once on another machine. Please help me, it is a critical matter for me.

**Comment by MantasTumenas** [comment]
Ok, I need zabbix agent log, zabbix conf file, user parameters file and printscreen, from host discovery tabs and host items tabs

**Comment by MantasTumenas** [comment]
ioannis-pan issue moved to a separate issue. Original issue closed due to no response from UneasyTheWay
